apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -743,6 +743,16 @@ export class RushConfiguration {
     return this._versionPolicyConfiguration;
   }
 
+  public getCurrentProjectFromPath(currentFolderPath: string): RushConfigurationProject | undefined {
```

Comment:
>getCurrentProjectFromPath [](start = 9, length = 25)

`getProjectFromPath`? It doesn't necessarily need to be the current project or the current path.